assets/js/controllers/settings/sweepAll.controller.js

```diff
@@ -0,0 +1,33 @@
+angular
+  .module('walletApp')
+  .controller('SweepAllController', SweepAllController);
+
+function SweepAllController($scope, $q, $timeout, $translate, Wallet, Alerts) {
+  $scope.alerts = [];
+  $scope.defaultIdx = Wallet.my.wallet.hdwallet.defaultAccountIndex;
+  $scope.addresses = Wallet.legacyAddresses().filter(a => a.active && !a.isWatchOnly);
```

Comment:
Should also add language as to whether or not the sweep will happen in one big transaction or one transaction per address.